GVRf/Framework/src/org/gearvrf/scene_objects/GVRTextViewSceneObject.java

```diff
@@ -39,7 +39,32 @@
 import android.widget.TextView;
 
 public class GVRTextViewSceneObject extends GVRSceneObject {
```

Comment:
Just a quick note for the future,
Multiple text views under the default Linear Layout for some cases,
public GVRTextViewSceneObject(…,
            TextView[] textViews) { }
but it will require changes from the single text view based methods and its full view size.